rust/agents/validator/src/submit.rs

```diff
@@ -48,6 +48,18 @@ impl ValidatorSubmitter {
         } else {
             Some(self.reorg_period)
         };
+
+        // Sign and post the validator announcement
+        let announcement = Announcement {
+            mailbox_address: self.mailbox.mailbox().address(),
+            mailbox_domain: self.mailbox.mailbox().domain().id(),
+            storage_metadata: self.checkpoint_syncer.announcement_metadata(),
+        };
+        let signed_announcement = announcement.sign_with(self.signer.as_ref()).await?;
```

Comment:
Genius idea to just write it to the syncer, though I think it might be nice to still just log the signed announcement here to avoid the operator needing to look it up on the s3 bucket?

Reply:
It seems easier to me to look it up in the bucket?